src/openforms/js/components/admin/Keyboard.js

```diff
@@ -0,0 +1,25 @@
+export const saveAndContinueButtonQuery = "form input[type=submit][name=_continue]";
+
+const saveAndContinueHandler = (event) => {
+    // because of the way React initializes we run the querySelector when the buttons is pressed
+    const button = document.querySelector(saveAndContinueButtonQuery);
+    if (!button) {
+        console.warn("Cannot find 'Save & Continue' button");
+    } else {
+        event.preventDefault();
+        button.click();
+    }
+}
+
+const enableKeyboardShortcuts = () => {
+    window.addEventListener('DOMContentLoaded', (e) => {
+        document.addEventListener('keydown', event => {
+            // add a Ctrl+S keyboard listener as trigger for Django's standard "Save & Continue" button
+            if (event.ctrlKey && event.key === 's') {
```

Comment:
I think this doesn't capture command key on Mac, see https://stackoverflow.com/a/3922353 - I think you also want to check `event.metaKey`